### PR TITLE
Update pull_request_template.md

### PR DESCRIPTION
**Pull request by kuznetsov**
#### Description

Remove screenshots table, turn Jira ticket to hyperlink, reduced titles text size

### CODE REVIEW

**Review by kabramovp2p — APPROVED** [review]

**Comment by kabramovp2p** [comment]
Nice 👍

**Comment by 1sakov** [comment]
Why are you removing placeholders for screenshots?

**Comment by kuznetsov** [comment]
> Why are you removing placeholders for screenshots?

Looks like they're used less frequently then don't, so we often should remove template text manually

File: .github/pull_request_template.md [hunk 1]
```diff
@@ -1,13 +1,7 @@
-## Jira Ticket (Mandatory)
+#### Ticket
 
-https://p2pvalidator.atlassian.net/browse/
+[PWN-XXXX](https://p2pvalidator.atlassian.net/browse/XXXX)
```
Comment (gslevinkov) [inline .github/pull_request_template.md:3]: I liked the full hyperlink more

Comment (kuznetsov) [reply .github/pull_request_template.md:3]: Why? 😶

**Comment by 1sakov** [comment]
> > Why are you removing placeholders for screenshots?
> 
> Looks like they're used less frequently then don't, so we often should remove template text manually

If there no screenshots, you don't need to remove placeholders. 


In general we should practice more using screenshots and we'll need to create placeholders for screenshots manually then, that's more work

File: .github/pull_request_template.md (hunk 1 above)
Comment (1sakov) [reply .github/pull_request_template.md:3]: The ticket number is redundant here. We use it in the title. 
In the description we just need to switch to Jira quickly, therefore there's only one action from develop: paste the ticket number which he used in the PR title

**Pull request closed without merging**